src/modelframe.jl

```diff
@@ -87,7 +87,7 @@ function check_non_redundancy!(trms::Terms, df::AbstractDataTable)
                 dropped = trms.factors[:,i_term]
                 dropped[i_eterm] = 0
 
-                if findfirst(encountered_columns, dropped) == 0
+                if findfirst(equalto(encountered_columns), dropped) == 0
```

Comment:
This doesn't work on 0.6. Use `x -> x == encountered_columns` instead.

Reply:
I don't think there will be a compatible `DataFrames` / `StatsModels` version for Julia 0.6. Should the package still provide support for it? I believe 1.0 would be 0.7 without the depreciations mostly so it might be better to adjust it now.

Reply:
Yes, there will be a 0.6 version AFAIK. Anyway the version I proposed will work everywhere, it's just slightly more verbose.